apps/api/src/app/environments/environments.controller.ts

```diff
@@ -99,6 +107,44 @@ export class EnvironmentsController {
     );
   }
 
+  @Put('/:envId')
```

Comment:
```suggestion
  @Put('/:environmentId')
```

Could we rename this to be `environmentId`?